integrations-and-supported-tools/xgboost/scripts/Neptune_XGBoost_sklearn_api.py

```diff
@@ -38,7 +38,6 @@
     X_train,
     y_train,
     early_stopping_rounds=30,
-    eval_metric=["mae", "rmse"],
```

Comment:
What was the issue here?

Reply:
I'm curious as well

Reply:
`eval_metrics` were being passed twice.. first during initialization (L34), and then again during `fit()` here.
This led to the 

> ValueError: 2 different `eval_metric` are provided.